demo/kserve/built-tip.md

```diff
@@ -3,37 +3,35 @@
 Caikit-TGIS image always requires a Caikit-formatted model. Below are the instructions from converting a model to a Caikit-formatted one.
 
 1. Clone the model repository (or have the model folder in a directory). In the below example, Bloom-560m model repo is cloned.
-~~~
+
+```bash
 yum -y install git git-lfs
```

Comment:
Is there a macos equivalent ? Should we mention if you are a "MacOS" user you are SOL ?

Reply:
Well, I'm assuming the equivalent is `brew install git-lfs`

Reply:
https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=mac